scripts/ag

```diff
@@ -54,5 +54,30 @@ def persample_json(table, output):
             fp.write(json.dumps(tree, indent=2))
 
 
+@ag.command()
+@click.argument('accession-metadata', required=True, nargs=-1)
+def sample_to_accession(accession_metadata):
+    """Dump {accession: [sample]} and {sample: accession}"""
```

Comment:
Hmmm, I guess I wasn't familiar with that notation, if it's standard
then I should revise my ways, so yeah just go with whatever is standard
(which seems like the current approach).

On (Jul-31-15|15:59), Daniel McDonald wrote:
> > @@ -54,5 +54,30 @@ def persample_json(table, output):
> >              fp.write(json.dumps(tree, indent=2))
> >  
> >  
> > +@ag.command()
> > +@click.argument('accession-metadata', required=True, nargs=-1)
> > +def sample_to_accession(accession_metadata):
> > +    """Dump {accession: [sample]} and {sample: accession}"""
> 
> Isn't that one of the normal notations to see? Like `[int]` implies `list` of `int`. Can change if you'd like tho
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/American-Gut/pull/154/files#r36024162